tests/testthat/_snaps/create_model_task.md

```diff
@@ -87,6 +87,134 @@
       attr(,"schema_id")
       [1] "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.1/tasks-schema.json"
 
+---
+
+    Code
+      create_model_task(task_ids = create_task_ids(create_task_id("origin_date",
+        required = NULL, optional = c("2023-01-02", "2023-01-09", "2023-01-16"),
+        branch = "br-v3.0.0"), create_task_id("location", required = "US", optional = c(
+        "01", "02", "04", "05", "06"), branch = "br-v3.0.0"), create_task_id(
+        "horizon", required = 1L, optional = 2:4, branch = "br-v3.0.0")),
+      output_type = create_output_type(create_output_type_mean(is_required = TRUE,
+        value_type = "double", value_minimum = 0L, branch = "br-v3.0.0"),
+      create_output_type_sample(is_required = FALSE, output_type_id_type = "character",
+        max_length = 5L, min_samples_per_task = 70L, max_samples_per_task = 100L,
+        compound_taskid_set = c("horizon", "origin_date"), value_type = "double",
+        value_minimum = 0L, value_maximum = 1L, branch = "br-v3.0.0")),
+      target_metadata = create_target_metadata(create_target_metadata_item(target_id = "inc hosp",
+        target_name = "Weekly incident influenza hospitalizations", target_units = "rate per 100,000 population",
+        target_keys = NULL, target_type = "discrete", is_step_ahead = TRUE,
+        time_unit = "week", branch = "br-v3.0.0")))
+    Output
+      $task_ids
+      $task_ids$origin_date
+      $task_ids$origin_date$required
+      NULL
+      
+      $task_ids$origin_date$optional
+      [1] "2023-01-02" "2023-01-09" "2023-01-16"
+      
+      
+      $task_ids$location
+      $task_ids$location$required
+      [1] "US"
+      
+      $task_ids$location$optional
+      [1] "01" "02" "04" "05" "06"
+      
+      
+      $task_ids$horizon
+      $task_ids$horizon$required
+      [1] 1
+      
+      $task_ids$horizon$optional
+      [1] 2 3 4
+      
+      
+      
+      $output_type
+      $output_type$mean
+      $output_type$mean$output_type_id
+      $output_type$mean$output_type_id$required
+      [1] NA
+      
+      $output_type$mean$output_type_id$optional
+      NULL
+      
+      
+      $output_type$mean$value
+      $output_type$mean$value$type
+      [1] "double"
+      
+      $output_type$mean$value$minimum
+      [1] 0
+      
+      
+      
+      $output_type$sample
+      $output_type$sample$output_type_id_params
+      $output_type$sample$output_type_id_params$is_required
+      [1] FALSE
+      
+      $output_type$sample$output_type_id_params$output_type_id_type
```

Comment:
related to comment earlier, I think we want this property to be called `$output_type$sample$output_type_id_params$type`, righ?

Reply:
Yup! resolved in 05245e748816e78d08fa20de9a052800bb77e44e